rickshaw.js

```diff
@@ -2458,6 +2458,10 @@ Rickshaw.Graph.RangeSlider = Rickshaw.Class.create({
 
 					graph.window.xMin = ui.values[0];
 					graph.window.xMax = ui.values[1];
+
+					console.log("graph.window.xMin: "+graph.window.xMin);
+					console.log("graph.window.xMax: "+graph.window.xMax);
```

Comment:
I case they do get around to looking at this, might want to remove the console.log entries.